### PR TITLE
Iris 356 branch project cypress

### PR DESCRIPTION
**Pull request by paweltomaszewskisaucelabs**

### CODE REVIEW

File: cypress/cypress.config.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ export default defineConfig({
   e2e: {
     saucelabs: {
       buildName: 'SauceDemo - Cypress',
+      branch: 'main',
+      project: 'CY examples',
```
Comment (ebekebe) [inline cypress/cypress.config.js:9]: Alternative to above: use repo name.

```suggestion
      buildName: 'Cypress job on git sha abcdef',
      branch: 'main',
      project: 'saucelabs/visual-examples',
```

File: cypress/README.md [hunk 2]
```diff
@@ -151,20 +158,22 @@ export SAUCE_ACCESS_KEY=__YOUR_SAUCE_ACCESS_KEY__
...
 Example
+
 ```javascript
 export default defineConfig({
   e2e: {
     [...]
     saucelabs: {
       buildName: 'SauceDemo - Cypress',
+      branch: 'main',
+      project: 'CY examples'
```
[the remark this answers is not in the document]
Comment (paweltomaszewskisaucelabs) [reply cypress/README.md:176]: I think this should stay just as a simple example. I would recommend to create separate example or instruction how to setup ci

File: cypress/cypress.config.js (hunk 1 above)
Comment (paweltomaszewskisaucelabs) [reply cypress/cypress.config.js:9]: I did it the same way as wdio. I recommend to create a ticket for renaming and unification for all client bindings.

Comment (paweltomaszewskisaucelabs) [reply cypress/cypress.config.js:9]: @ebekebe I think we should proceed with this mr.

**Pull request closed without merging**